saos-enrichment/src/main/java/pl/edu/icm/saos/enrichment/upload/EnrichmentTagUploadController.java

```diff
@@ -92,6 +99,8 @@ private void asyncProcessUploadedEnrichmentTags() {
 		new Thread(new Runnable() {
 			        		public void run() {
 			        			uploadEnrichmentTagProcessor.processUploadedEnrichmentTags();
+			        			
+			        			runTagPostUploadProcessingJob();
```

Comment:
I think it would be better to move the job execution to processUploadedEnrichmentTags.